src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -311,6 +321,11 @@ public async Task<TokenResponse> RedeemInteractionCodeAsync(IIdxContext idxConte
                 response.Initialize(this, null, data, _logger);
                 return response;
             }
+            catch (RedeemInteractionCodeException redeemInteractionCodeException)
+            {
+                LogError(redeemInteractionCodeException);
+                throw redeemInteractionCodeException;
```

Comment:
It is usually not a recommended practice to catch and rethrow the same exception (if we're not doing anything to handle) because exceptions are expensive.
That being said if we still need to rethrow the same exception we should use just `throw;` because that way the stack trace is preserved. I like[ this article](https://hamidmosalla.com/2018/02/28/net-exception-best-practices/) for short guidance on the topic: